index.d.ts

```diff
@@ -339,4 +343,41 @@ declare module 'binance-api-node' {
         isOrderWorking: boolean;
         isBuyerMaker: boolean;
     }
+
+    interface MyTrade {

```

Comment:
Please rename it to `Trade` to match the naming schema of: https://github.com/binance-exchange/binance-java-api/blob/master/src/main/java/com/binance/api/client/domain/account/Trade.java

Reply:
The name `Trade` is already used for websocket trade events.